flytekit/taskplugins/hive/task.py

```diff
@@ -10,20 +11,29 @@
 from flytekit.types.schema import FlyteSchema
 
 
-class HiveTask(SQLTask):
+@dataclass
+class HiveConfig(object):
+    cluster_label: str
+    tags: Optional[List[str]] = None
```

Comment:
Can you add a comment on what these params are for?

Reply:
doing it now